packages/unmock-core/src/backend/index.ts

```diff
@@ -92,7 +93,9 @@ interface IBypassableSocket extends net.Socket {
 }
 
 export default class NodeBackend {
-  private serviceStore: ServiceStoreType = {};
+  private static dummyStore = { services: {} };
+
+  private serviceStore: ServiceStore | undefined;
```

Comment:
Should this rather be always defined but possibly empty?

Reply:
It should - it would mean refactoring the backend so that it creates its own service store (where now it's a part of the `responseCreatorFactory` function, and it's assigned later on).